app/config/settings.py

```diff
@@ -613,7 +613,7 @@ def strtobool(val) -> bool:
 # CIRRUS Is an external application that can view images
 CIRRUS_APPLICATION = "https://apps.diagnijmegen.nl/Applications/CIRRUSWeb_master_98d13770/#!/?workstation=BasicWorkstation"
 CIRRUS_BASE_IMAGE_QUERY_PARAM = "grand_challenge_image"
-CIRRUS_ANNOATION_QUERY_PARAM = "grand_challenge_overlay"
```

Comment:
Thanks for correcting the typo, but could you check for other usages in the codebase?

Reply:
Yes I did. I checked by performing a find all.. on the setting name. It is not used anywhere else yet.

Reply:
Check your query - it's used in `datasets/models.py`

Reply:
Hmm. I'm not sure how I missed that. Maybe I used the fixed name to search for it. 😅 
Anyway, fixed it now.